pkgs/applications/misc/golden-cheetah/default.nix

```diff
@@ -16,18 +16,18 @@ let
   };
 in mkDerivation rec {
   pname = "golden-cheetah";
-  version = "3.5";
+  version = "unstable-2021-05-01";
```

Comment:
```suggestion
  version = "3.5-DEV1804-1240-gc9e01353a";
```

Reply:
No, the latest tag is `3.6-DEV2106`, as written in https://github.com/NixOS/nixpkgs/pull/121373#issuecomment-876293264.

Reply:
I updated this to `3.6-DEV2107` (new release)